app/models/container_image.rb

```diff
@@ -92,9 +92,22 @@ def annotate_deny_execution(causing_policy)
     ext_management_system.annotate(
       "image",
       digest,
+      annotations
+    )
+  end
+
+  def annotate_allow_execution(causing_policy)
+    annotate_image({
+      "security.manageiq.org/successful-policy" => causing_policy,
+      "images.openshift.io/allow-execution"     => "true"
```

Comment:
Is `allow-execution` annotation even a thing?  Will openshift admission plugin look at it?  https://docs.openshift.com/container-platform/latest/admin_guide/image_policy.html gives me the impression only `deny-execution` is a thing, with value true or false.